nightly-examples/examples/echo.rs

```diff
@@ -0,0 +1,119 @@
+//! A "hello world" echo server [from Tokio][echo-example]
+//!
+//! This server will create a TCP listener, accept connections in a loop, and
+//! write back everything that's read off of each TCP connection.
+//!
+//! Because the Tokio runtime uses a thread pool, each TCP connection is
+//! processed concurrently with all other TCP connections across multiple
+//! threads.
+//!
+//! To see this server in action, you can run this in one terminal:
+//!
+//!     cargo +nightly run --example echo
+//!
+//! and in another terminal you can run:
+//!
+//!     cargo +nightly run --example proxy_server 127.0.0.1:8081
+//!
+//! Each line you type in to the `connect` terminal should be echo'd back to
```

Comment:
I think this line needs to be updated since there's no `connect` example. Maybe we should add one, instead of showing the proxy server? Or, we could suggest that folks use `telnet` or `netcat` to send data to the echo server through the proxy.

Reply:
The docstring in the `proxy_server` example already has the case of connecting to the `echo` server using `netcat` using the proxy server. I think for the echo server, simply using `netcat` should be enough.